go/oasis-node/cmd/common/grpc/grpc.go

```diff
@@ -73,6 +77,11 @@ func NewServerLocal(installWrapper bool) (*cmnGrpc.Server, error) {
 func NewClient(cmd *cobra.Command) (*grpc.ClientConn, error) {
 	addr, _ := cmd.Flags().GetString(CfgAddress)
 
+	if len(addr) > 0 && addr[0] == '/' {
```

Comment:
this also doesn't cover the case where if path is relative (i still think changing the `WaitForReady` in cli cases is a more correct fix)

Reply:
I included both now. If `addr` is the name of a local file, it prepends `unix:`. In either case, it never enables `WaitForReady` option. Let's see how tests react...